avocado/core/loader.py

```diff
@@ -694,8 +693,7 @@ def ignore_broken(klass, uid, params=None):
                                                 subtests_filter)
             else:
                 if os.access(test_path, os.X_OK):
-                    return self._make_test(test.SimpleTest,
-                                           pipes.quote(test_path))
+                    return self._make_test(test.SimpleTest, test_path)
```

Comment:
> The "foo bar" is a requirement. It's actually the primary usage.
Where do we state so? I'm not convinced. And it seems wrong and hacky. It's the first time I'm thinking this through, but right now we are also inconsistent, because on `master`, this does not work:
```
avocado run '/tmp/test script.sh'
```
while this works:
```
cd /tmp
avocado run 'test script.sh'
```
And none of them have anything to do with unicode. But `pipes.quotes()`, which is making the second example to work, is also breaking tests with unicode characters. I think we should remove this `pipes.quote()` and handle the spaces in test names properly, after making clear what are the requirements (like in a RFC).